docs/UserGuide.md

```diff
@@ -401,19 +401,18 @@ Example:
 Deletes a single session from an existing recurring tuition session of a specified student from TutorBuddy.
 Spawns two recurring sessions that will span the period exclusively before and after the deleted single session.
 
-Format: `delete_rec_session n/STUDENT_NAME i/SESSION_INDEX d/DATE t/TIME`
+Format: `delete_rec_session n/STUDENT_NAME i/SESSION_INDEX d/DATE`
 
 Arguments similar to `delete_session` command except the following:
 * `DATE` should be the date of the single session to be deleted
-* `TIME` should be the time of the single session to be deleted
 <div markdown="block" class="alert alert-info">
-:information_source: Note that the <code>DATE</code> and <code>TIME</code> should be a valid session date that belongs in the specified recurring session
+:information_source: Note that the <code>DATE</code> should be fall under a valid session date that belongs in the specified recurring session range
```

Comment:
Maybe removing the "be" from "should be fall under" would be better.

Reply:
Nice catch, pardon my carelessness. 🥴